.flowconfig

```diff
@@ -20,5 +20,6 @@ module.name_mapper='^lbry\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/lbry\1'
 module.name_mapper='^rewards\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/rewards\1'
 module.name_mapper='^modal\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/modal\1'
 module.name_mapper='^app\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/app\1'
+module.name_mapper='^native\(.*\)$' -> '<PROJECT_ROOT>/src/renderer/native\1'
```

Comment:
Is there a reason this was added?

Reply:
@seanyesmunt This was to resolve a flow error that was on [in line](https://github.com/amitnndn/lbry-app/blob/aa493f997bd99dd9ef620484bf3585d6604b4d0f/src/renderer/page/help/view.jsx#L6)